roofit/roofitcore/src/RooGlobalFunc.cxx

```diff
@@ -80,6 +79,30 @@ RooCmdArg processFlatMap(const char *name, Func_t func, Detail::FlatMap<Key_t, V
    return container;
 }
 
+int interpretString(std::string const &s)
+{
+   return gInterpreter->ProcessLine(s.c_str());
+}
+
+Color_t interpretColorString(std::string const &color)
+{
+   using Map = std::unordered_map<std::string, std::string>;
+   // Color dictionary to define matplotlib conventions
+   static Map colorMap{{"r", "kRed"},   {"b", "kBlue"},  {"g", "kGreen"},   {"y", "kYellow"},
+                       {"w", "kWhite"}, {"k", "kBlack"}, {"m", "kMagenta"}, {"c", "kCyan"}};
+   auto found = colorMap.find(color);
+   return gInterpreter->ProcessLine((found == colorMap.end() ? color : found->second).c_str());
```

Comment:
I don't immediately understand the necessity of this. IIUC, this will end up looking like, for example
```
gInterpreter->ProcessLine("kRed");
```

Can't we just get a one to one mapping between the color string and the ROOT color number without asking the interpreter?

Reply:
That is true, it's a little optimization in case one uses the matplotlib color codes. I'll update that!